src/OpenToolkit.Mathematics/Geometry/Box2.cs

```diff
@@ -195,10 +176,10 @@ public override int GetHashCode()
         {
             unchecked
             {
-                var hashCode = Left.GetHashCode();
-                hashCode = (hashCode * 397) ^ Right.GetHashCode();
-                hashCode = (hashCode * 397) ^ Top.GetHashCode();
-                hashCode = (hashCode * 397) ^ Bottom.GetHashCode();
+                var hashCode = Min.X.GetHashCode();
+                hashCode = (hashCode * 397) ^ Min.Y.GetHashCode();
+                hashCode = (hashCode * 397) ^ Max.X.GetHashCode();
+                hashCode = (hashCode * 397) ^ Max.Y.GetHashCode();
                 return hashCode;
```

Comment:
Just use `Min.GetHashCode()` and `Max.GetHashCode()` - by the time we get to Box3, this will save more than a few lines.

Reply:
Also, `.GetHashCode()` can be omitted for integer types, since all it does on integers is return the integer itself anyway.